### PR TITLE
Adds a verb to check one's own playtime.

### PR DESCRIPTION
**Pull request by Birdtalon**
A verb to check one's own playtime on the server. If it requires fix tokens I can quote them.

🆑 Birdtalon
add: Verb to check your own playtime.
/🆑

### CODE REVIEW

**Comment by variableundefined** [comment]
Free

**Comment by variableundefined** [comment]
Need head approval for this one, so 24 hours.

**Comment by Ty-Omaha** [comment]
>YES no more mhelps for this!

mentorhelp from me: "How do I check my playtime?"

sucker

**Comment by variableundefined** [comment]
I wonder if the whole window should be made visible to the player instead.

File: code/game/jobs/job_exp.dm [hunk 1]
```diff
@@ -37,6 +37,17 @@ var/global/list/role_playtime_requirements = list(
...
+// Client Verbs
+
+/client/verb/cmd_check_own_playtime()
+	set category = "Special Verbs"
+	set name = "Check my playtime"
+
+	if(!config.use_exp_restrictions)
+		return to_chat(src, "<span class='warning'>Playtime tracking is not enabled.</span>")
+
+	to_chat(src, "<span class='notice>Your playtime is [src.get_exp_living()].</span>")
```
Comment (KasparoVy) [inline code/game/jobs/job_exp.dm]: Implied `src`? :thinking:

File: code/game/jobs/job_exp.dm [hunk 2]
```diff
@@ -37,6 +37,17 @@ var/global/list/role_playtime_requirements = list(
...
 )
 
+// Client Verbs
+
+/client/verb/cmd_check_own_playtime()
+	set category = "Special Verbs"
+	set name = "Check my playtime"
+
+	if(!config.use_exp_restrictions)
+		return to_chat(src, "<span class='warning'>Playtime tracking is not enabled.</span>")
```
Comment (Crazylemon64) [inline code/game/jobs/job_exp.dm]: These should be separate lines

File: code/game/jobs/job_exp.dm (hunk 1 above)
Comment (Crazylemon64) [reply code/game/jobs/job_exp.dm]: Indeed, use `get_exp_living()` without a `src` access

**Comment by KasparoVy** [comment]
Nice :sunglasses: :+1:

**Comment by Kyep** [comment]
No real disadvantage to showing them their entire playtime report.
Would actually help people in some cases, too, as it would display their progress towards unlocking all the time-locked job roles.
Plus it would tell them things like what % of the time they're in a special role. (ie: are antag)

Once the freeze is over, I'll likely put up a separate PR that makes the playtime report in general easier to read.

**Comment by variableundefined** [comment]
Approved and 24 hours passed., so why not.

**Pull request merged**